### DISCUSSION TITLE
Space after the end of sentences truncated

### DISCUSSION THREAD

**Discussion by AGI-chandler** [community/community#45113] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Whenever I write something here, it always removes the extra space at the end of a sentence.  Actually, I notice it other places too outside of Github, and I notice it too even when reading static text on other sites that others have written.  Is this something caused by my browser (Firefox)?  or is it caused by my environment (Gnome)?  or is it an HTML issue?  or is it specific to Github (and the other places I've noticed it)?  or maybe it's a mixture of these?  As you can see, I figured a workaround for things I write here (may work other places I write as well), by converting the spaces to non-breaking spaces, but it involves copy/pasting and running the text through a script.  Personally, I don't like it, so I'm just trying to wrap my head around the source of it.  Thanks!

**Comment by jge162** [comment, marked as answer]
@AGI-chandler Some browsers and text editors may automatically remove extra spaces at the end of sentences or lines. But you can control with HTML or CSS code on the website you are viewing. 

Added screen shot below. Code block not working.

**Comment by jge162** [comment]
![EBF16AF3-3CF7-4122-B478-7D3E40B5BE4C](https://user-images.githubusercontent.com/31228460/213970613-4ccf2dc1-64e9-4968-9a18-b43ebf5bb089.jpeg)

**Reply by jge162** [reply]
’’’html
<p>This text contains a non-breaking space at the end: "Hello, World!&nbsp;"</p>

**Comment by jge162** [comment]
![29CF474F-EF4C-4AFF-9404-EB6470B06D76](https://user-images.githubusercontent.com/31228460/213971179-a3d9dd5c-ad0d-4ef2-8dbd-be6d83b6d58f.jpeg)
